Godeps/Godeps.json

```diff
@@ -2535,27 +2535,27 @@
 		},
 		{
 			"ImportPath": "github.com/libopenstorage/openstorage/api",
-			"Rev": "6e787003b91ddba85f108b8aede075b1af0d3606"
+			"Rev": "a53f5e5662367da02b95470980c5dbaadfe96c99"
```

Comment:
I might have understood the meaning of "carry" differently.
I updated this vendor'ed file to that particular commit. Should the commit message then be different?